### ISSUE TITLE
support custom metadata blocks

### ISSUE THREAD

**Issue by pdurbin** [IQSS/dataverse-ansible#133]
Similar to how we support ssh keys as of #62 I've been thinking it would be nice to support custom metadata blocks.

I think for now just specifying URLs like this should be fine:

(That custom metadata block is from https://github.com/IQSS/dataverse/issues/6289 by the way.)

```
custom_metadata_blocks:
  enabled: true
  urls:
    - https://github.com/IQSS/dataverse/files/3744336/codemeta.tsv.txt
```

The other metadata blocks are loaded by setup-all.sh here...

https://github.com/IQSS/dataverse-ansible/blob/309f47f2794e7533f52496fa00dabad14a6a3f40/tasks/dataverse-postinstall.yml#L24

... so I was thinking that it would make sense to load the extra blocks just after that. Here's the syntax to use:

`curl http://localhost:8080/api/admin/datasetfield/load -H "Content-type: text/tab-separated-values" -X POST --upload-file custom1.tsv`

Afterward, Solr needs to be updated. I was thinking we could call `updateSchemaMDB.sh` which is documented at http://guides.dataverse.org/en/4.18/admin/metadatacustomization.html#updating-the-solr-schema

As of this writing it can be found here in the main repo:

    conf/solr/7.3.1/updateSchemaMDB.sh

All of this will make it easier to test custom metadata blocks. I've also asked for this feature at https://github.com/IQSS/dataverse-kubernetes/issues/85 if it makes sense to compare notes! 😄

**Comment by pdurbin** [comment]
@donsizemore it works! I just merged pull request #135 . Thanks!

I specified two custom metadata blocks like this:

```
    user: counter
    year_month: "2018-05"
  custom_metadata_blocks:
    enabled: true
    urls:
      - https://github.com/IQSS/dataverse/files/3744336/codemeta.tsv.txt
      - https://raw.githubusercontent.com/IQSS/dataverse/develop/scripts/api/data/metadatablocks/customMRA.tsv
  db:
    name: dvndb
```

And they were loaded. Here's a screenshot.

<img width="1552" alt="Screen Shot 2019-11-18 at 4 52 32 PM" src="https://user-images.githubusercontent.com/21006/69097220-1d9ab500-0a24-11ea-9b1c-33b860a33399.png">

Then I created a dataset with the MRA metadata block and the MRA field was indexed just fine, shown in the facet below.

<img width="1552" alt="Screen Shot 2019-11-18 at 4 56 32 PM" src="https://user-images.githubusercontent.com/21006/69097385-7702e400-0a24-11ea-9724-a2925eb0b200.png">

Thanks!

**Comment by pdurbin** [comment]
@donsizemore I just gave you and @poikilotherm a shout out in this thread called "custom metadata blocks now easier to spin up and evaluate" - https://groups.google.com/d/msg/dataverse-community/uKretKox_io/4FyPVAMYBgAJ

Thanks again! 🎉 🎉 🎉
